cloudant-client/src/test/java/com/cloudant/api/query/IndexLifecycleTest.java

```diff
@@ -24,22 +24,20 @@
 import com.cloudant.client.api.query.TextIndex;
 import com.cloudant.client.api.query.Type;
 import com.cloudant.test.main.RequiresCloudant;
-import com.cloudant.tests.util.CloudantClientResource;
-import com.cloudant.tests.util.DatabaseResource;
+import com.cloudant.tests.extensions.CloudantClientExtension;
+import com.cloudant.tests.extensions.DatabaseExtension;
 
-import org.junit.After;
-import org.junit.Before;
-import org.junit.Rule;
-import org.junit.Test;
-import org.junit.experimental.categories.Category;
-import org.junit.rules.RuleChain;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
 
 import java.util.Collections;
 import java.util.List;
 
 
 // This really requires Couch2.0 + text index support, but we don't have a way of expressing that
-@Category(RequiresCloudant.class)
+@RequiresCloudant
```

Comment:
Probably best done in a follow-on PR. I wanted to get the tags exactly the same so that I could check each of the matrix builds executes the correct number of tests, before thinking about changing the tags.